kratos/mpi/python/add_distributed_sparse_matrices_to_python.cpp

```diff
@@ -23,8 +23,7 @@
 #include "containers/distributed_csr_matrix.h"
 #include "add_distributed_sparse_matrices_to_python.h"
 #include "mpi/utilities/amgcl_distributed_csr_spmm_utilities.h"
-
-
+#include <pybind11/operators.h>
```

Comment:
This should be in External includes

Reply:
i removed it all alltogether. It looks like it requires a more modern pybind than we have (or a higher c++ version)